README.md

```diff
@@ -7,7 +7,7 @@ This is a Python microservice created using FastAPI and requires a MongoDB insta
 ### Prerequisites
 
 - Docker and Docker Compose installed (if you want to run the microservice inside Docker)
-- Python 3.10 (or above) and MongoDB 7.0 installed on your machine (if you are not using Docker)
+- Python 3.12 and MongoDB 7.0 installed on your machine (if you are not using Docker)
```

Comment:
Should the `requires-python`  in the `pyproject.toml` be updated as well if we are saying its a requirement in the readme?

Reply:
Ah yes, I completely forgot about this, apologies..